ci-templates/native-build-steps.yaml

```diff
@@ -44,4 +44,4 @@ jobs:
       inputs:
         goals: 'install'
         mavenOptions: $(MAVEN_OPTS)
-        options: '-pl integration-tests/${{ join('',integration-tests/'', parameters.modules) }} -B --settings azure-mvn-settings.xml -Dnative-image.docker-build -Dtest-postgresql -Dtest-elasticsearch -Dtest-keycloak -Ddocker-keycloak -Dtest-dynamodb -Dtest-mysql -Dnative-image.xmx=6g -Dnative -Dno-format'
+        options: '-pl integration-tests/${{ join('',integration-tests/'', parameters.modules) }} -B --settings azure-mvn-settings.xml -Dnative-image.docker-build -Dtest-postgresql -Dtest-elasticsearch -Dtest-keycloak -Pdocker-keycloak -Dtest-dynamodb -Dtest-mysql -Dnative-image.xmx=6g -Dnative -Dno-format'
```

Comment:
the oidc tests use `docker-keycloak` as the profile name, the property to control it is `docker`. It seems to me that `-Ddocker-keycloak` was a typo here but maybe there's more magic here...